### DISCUSSION TITLE
Is ter-132b or other bold font supported in zfsbootmenu?

### DISCUSSION THREAD

**Discussion by mitosu-me** [zbm-dev/zfsbootmenu#316] [General]
ter-132n is a little too thin. My current consolefont is ter-d32b that use by my virtual console.

**Comment by ahesford** [comment]
Most definitely. I use ter-132b in my own images.

If you are using the prebuilt EFI executable, you can modify its command line with the [zbm-efi-kcl](https://github.com/zbm-dev/zfsbootmenu/blob/master/bin/zbm-efi-kcl) and run

    zbm-efi-kcl -e /path/to/zbm.efi

add `rd.vconsole.font=ter-132b` to any command line that appears in your editor, save and quit.

**Comment by mitosu-me** [comment]
Is terminus-font include in prebuild image or I have to install terminus-font in my system?

**Reply by zdykstra** [reply]
It's included in the prebuilt images, and can be enabled as previously described.
